common/containers/ParitySignerDecrypt/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import translate from 'translations';
+import QrSigner from '@parity/qr-signer';
+import { isValidETHAddress } from 'libs/validators';
+import { ParitySignerWallet } from 'libs/wallet';
+import {
+  setWalletQrTransaction,
+  TSetWalletQrTransaction,
+  finalizeQrTransaction,
+  TFinalizeQrTransaction
+} from 'actions/wallet';
+import './index.scss';
+import AppStoreBadge from 'assets/images/mobile/app-store-badge.png';
+import GooglePlayBadge from 'assets/images/mobile/google-play-badge.png';
+
+interface Props {
+  setWalletQrTransaction: TSetWalletQrTransaction;
+  finalizeQrTransaction: TFinalizeQrTransaction;
+  onUnlock(param: any): void;
+}
+
+class ParitySignerDecrypt extends PureComponent<Props> {
+  public render() {
+    return (
+      <div className="ParitySignerUnlock">
+        <div className="qr-bounds">
+          <QrSigner
+            size={300}
+            scan={true}
+            onScan={this.unlockAddress}
+            styles={{ display: 'inline-block' }}
+          />
+        </div>
+        <p>{translate('ADD_PARITY_2')}</p>
```

Comment:
At some point we should put together a help article that explains this process, or have some additional descriptive copy here.

Reply:
We are working on a proper help page that describes how to use the signer, I'll add a link to it here once it's done (ideally before publicity but can be done after the PR is merged).